### PR TITLE
[Explorer] search aptos names

### PR DESCRIPTION
**Pull request by zihan-aptos**
Screen Shot 2022-10-31 at 11 43 27 AM

### CODE REVIEW

**Review by msavir — APPROVED** [review]

File: src/api/hooks/useGetANS.ts [hunk 1]
```diff
@@ -59,3 +59,26 @@ export function useGetAddressFromName(name: string) {
...
+export async function getAddressFromName(
+  name: string,
+  network: NetworkName,
+): Promise<string | undefined> {
+  const addressUrl = getFetchAddressUrl(network, name);
+  if (addressUrl === undefined) {
+    return undefined;
+  }
+
+  const addressResponse = await fetch(addressUrl);
```
Comment (msavir) [inline src/api/hooks/useGetANS.ts:72]: should we add `catch`?

Comment (zihan-aptos) [reply src/api/hooks/useGetANS.ts:72]: Hmm... I can't think of a case when it'll throw error, unless the url has changed.

Comment (zihan-aptos) [reply src/api/hooks/useGetANS.ts:72]: Oh and the upper level where this function is called has try catch. So it should be safe.

Comment (msavir) [reply src/api/hooks/useGetANS.ts:72]: if the server is down for some reason...?

**Pull request merged**